.devcontainer/scripts/start-netbox.sh

```
@@ -10,33 +10,79 @@ fi
+fi
+
+# Activate NetBox virtual environment
 source /opt/netbox/venv/bin/activate
+
+# Navigate to NetBox directory
 cd /opt/netbox/netbox
 
+# Start RQ worker in background
+echo "⚙️  Starting RQ worker..."
+(
+  source /opt/netbox/venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: /opt/netbox/venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)